autopush/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.2.3'
+__version__ = '1.3.0dev'
```

Comment:
I think I'd prefer that. Less chance of forgetting later, we're already on the next version, and we have a single track for code right now anyway.